src/test/store/actions.test.js

```diff
@@ -16,9 +16,44 @@ import {
   changeShowJsTracerSummary,
   changeShowUserTimings,
 } from '../../actions/profile-view';
-import { getProfileFromTextSamples } from '../fixtures/profiles/processed-profile';
+import {
+  getProfileFromTextSamples,
+  getProfileWithMarkers,
+  getUserTiming,
+} from '../fixtures/profiles/processed-profile';
 import { selectedThreadSelectors } from '../../selectors/per-thread';
 
+function setupUserTimings() {
+  // Approximately generate this type of graph with the following user timings.
+  //
+  // [renderFunction---------------------]
+  //   [componentA---------------------]
+  //     [componentB----]  [componentD]
+  //      [componentC-]
+
+  return getProfileWithMarkers([
+    getUserTiming('renderFunction', 0, 10),
+    getUserTiming('componentA', 1, 8),
+    getUserTiming('componentB', 2, 4),
+    getUserTiming('componentC', 3, 1),
+    getUserTiming('componentD', 7, 1),
+  ]);
+}
+
+function setupSamples(samples) {
```

Comment:
Optional: I think these both could go into the `describe('selectors/getCombinedTimingRows` block so that the global scope is a bit cleaner. In addition, this function could be simplified to not take parameters. It's pretty much just running `getProfileFromTextSamples` already, which would be better to use rather than overload this one.